src/events/message.rs

```diff
@@ -59,6 +65,46 @@ pub async fn message(ctx: client::Context, msg: Message) -> Result<()> {
     Ok(())
 }
 
+async fn handle_highlighting(ctx: &client::Context, msg: &Message) -> Result<()> {
+    let db = ctx.get_db().await;
+    let highlights = db.get_highlights().await?;
+
+    for i in &mut msg
+        .content
+        .split_whitespace()
+        .into_iter()
+        .map(|p| p.to_string())
+    {
+        // checks if the database contains our word
+        if let Some(m) = highlights.get(&i.to_string()) {
+            // iterates over each user that subscribed to that highlight and send them a message
+            for x in m {
+                let channel = msg.channel(&ctx).await.context("Couldn't get channel")?.guild().context("Couldn't get server")?.name;
+                let guild = msg.guild(&ctx).await.unwrap().name;
```

Comment:
definitely do these outside of the for loop, lol. No need to load them again and again every time

Reply:
also no unwrap REEEEEEEEEEEEEEEEEEEEEEEEE
and just call it `guild_name` if you're only storing the guild name